x11-packages/xwallpaper/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/stoeckmann/xwallpaper
+TERMUX_PKG_DESCRIPTION="wallpaper setting utility for X"
+TERMUX_PKG_LICENSE="ISC"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.7.5
+TERMUX_PKG_SRCURL=https://github.com/stoeckmann/xwallpaper/archive/refs/tags/v0.7.5.tar.gz
+TERMUX_PKG_SHA256=d64b8bae1700835d1c0996b28ff0e9d4a93ead4f8698bbdb6acc19150537aa23
+TERMUX_PKG_DEPENDS="libpixman, xcb-util-image, libjpeg-turbo, libpng, libxpm"
+
+termux_step_pre_configure() {
+	export PKG_CONFIG_PATH=/data/data/com.termux/files/usr/lib/pkgconfig
```

Comment:
I do not believe this is needed. Also why the hardcode prefix?

Reply:
I was invoking `./build-package` with `-s` on clean environment and wondered why it does not work. Imma stupid, sorry.